sources/packages/backend/apps/api/src/services/user/user.model.ts

```diff
@@ -5,4 +5,6 @@ export interface UserLoginInfo {
   isActive: boolean;
   studentId?: number;
   identityProviderType?: IdentityProviders;
+  firstName?: string;
+  lastName: string;
```

Comment:
Where are these properties consumed?

Reply:
Yes, they aren't consumed anywhere. I had added they won't be used. Thanks!